packages/cli/openapi-parser/src/v3/converters/operation/convertStreamingOperation.ts

```diff
@@ -31,10 +35,191 @@ export function convertStreamingOperation({
                 nonStreaming: undefined,
             };
         }
-        case "streamCondition":
-            break;
+        case "streamCondition": {
+            const streamingOperation = convertHttpOperation({
+                operationContext: {
+                    ...operationContext,
+                    operation: {
+                        ...operationContext.operation,
+                        requestBody: convertRequestBodyToInlinedRequest({
```

Comment:
thoughts on renaming this to `createStreamingRequestBody({ context, requestBody, streamConditionProperty })`; 

`inlinedRequest` is more a fern term so figured we could remove that

Reply:
Totally fine with that, let's do it